frontend/kangra-earthquake-1905-explorer/style.css

```
@@ -0,0 +1,499 @@
+:root {
+    --bg: #101820;
+    --bg-alt: #16202b;
+    --surface: #1c2836;
+    --line: #314155;
+    --text: #eef3f8;
+    --muted: #b4c2d1;
+    --dim: #8a9aab;
+    --accent: #f59e0b;   /* Amber / Terracotta Warmth */
+    --accent-2: #dc2626; /* Warning Red */
+    --heading: "Playfair Display", Georgia, serif;
```

Stylelint (17.14.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)